### PR TITLE
 Add: theme, language select feature in editor page

### PR DESCRIPTION
**Pull request by ysm0622**
image

Added Theme and Language select features in Editor page with Carbon, CodeMirror References.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,10 +91,12 @@
     "passport-github": "^1.1.0",
     "puppeteer": "^1.8.0",
     "react": "^16.5.0",
+    "react-codemirror2": "^5.1.0",
     "react-dom": "^16.5.0",
     "redis": "^2.8.0",
     "styled-components": "^3.4.5",
     "styled-normalize": "^8.0.0",
+    "tohash": "^1.0.2",
```
Comment (gimdongwoo) [inline package.json]: Why `tohash ` is needed?

Comment (ysm0622) [reply package.json]: I used for convenience.

Comment (thomasJang) [reply package.json]: Why don't use rebase?

Comment (ysm0622) [reply package.json]: You mean git rebase?

Comment (siosio34) [reply package.json]: this merge request have many merge commit. i think one merge commit  to one merge request. so @thomasJang say why don't use rebase.  when you send merge request, i recommend "git pull --rebase origin master" in your branch.  this command your commit list merge and rebase easy before you send merge request.

**Pull request closed without merging**